.github/workflows/benchmark_wasm_client.yml

```diff
@@ -106,12 +106,12 @@ jobs:
       - name: Install web resources
         run: |
           make install_node
-          make install_chrome_browser
-          make install_chrome_web_driver
+          make install_firefox_browser
+          make install_firefox_web_driver
 
       - name: Run benchmarks
         run: |
-          make bench_web_js_api_parallel_chrome_ci
+          make bench_web_js_api_parallel_firefox_ci
```

Comment:
could we have both benchmarks ? at least initially to be able to compare ?

Reply:
I've put a mechanism to do so.